blackjack/console.py

```diff
@@ -0,0 +1,126 @@
+    initial_deal, has_blackjack, hit, get_winner
+
+import random
+
+
+def show_cards(state: GameState):
+    print("")
+    print(f'Player 1: {render_hand(state.player_hand)}')
+    print(f'The House: {render_dealer(state.dealer_hand)}\n')
+
+
+def display_winner(winner: Conclusion, state: GameState):
+    player_total = hand_total(state.player_hand)
+    house_total = hand_total(state.dealer_hand)
+    if winner is Conclusion.Push:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n PUSH!')
+        print("")
+        return
+    if winner is Conclusion.PlayerBj:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'BLACKJACK! {render_hand(state.player_hand)}= {player_total} \nYOU WIN!')
+        print("")
+        return
+    if winner is Conclusion.HouseBj:
+        print("")
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total}')
+        print(f'BLACKJACK! {render_hand(state.dealer_hand)}= {house_total} \nHouse wins!')
+        print("")
+        return
+    if winner is Conclusion.PlayerBust:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'You Bust! {render_hand(state.player_hand)}= {player_total} \nHouse Wins!')
+        print("")
+        return
+    if winner is Conclusion.HouseBust:
+        print("")
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total}')
+        print(f'House Busts: {render_hand(state.dealer_hand)}= {house_total} \nYou Win!')
+        print("")
+        return
+    if winner is Conclusion.HouseWin:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n House Wins!')
+        print("")
+        return
+    if winner is Conclusion.PlayerWin:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n YOU Win!')
+        print("")
+        return
+    raise ValueError(f'Winner Inconclusive for {winner}')
+
+
+def get_hit_or_stay(state: GameState) -> Play:
+    total = hand_total(state.player_hand)
+    s = input(f'Your Total is {total}, would you like to Hit? ("y" or "n"): ')
+    play = parse_play(s)
+    if play:
+        return play
+    print("Please choose 'y' or 'n'.")
+    return get_hit_or_stay(state)
+
+
+def check_play_again():
+    while True:
+        a = input("Would you like to play again? ('y' or 'n'): ")
+        print("")
+        if a == 'y':
+            return True
+        elif a == 'n':
+            print('Goodbye!')
+            exit()
+        else:
+            print('Please Choose "y" or "n"!')
+            continue
```

Comment:
Notice that this function only ever returns True; if that's the case, is the return value useful?

I will add that it's a little weird to use exit() in this function. You don't necessarily need to change that now, but it's worth thinking about what you expect this function to do, especially considering the return value.